vars/runTest.groovy

```diff
@@ -127,13 +132,14 @@ Map call(Map config = [:]) {
     }
 
     if (status != 'SUCCESS') {
+        String msg = ' See Junit report for details'
```

Comment:
Maybe something more generic like:

```suggestion
        String msg = ' See job Test Results for details'
```

Then a user doesn't have to know that the Test Results are JUnit.